esphome/components/esp32_can/esp32_can.cpp

```diff
@@ -0,0 +1,121 @@
+#include "esp32_can.h"
+#include "esphome/core/log.h"
+
+#include <driver/can.h>
```

Comment:
```suggestion
#ifdef ARDUINO_ARCH_ESP32
#include <driver/can.h>
#endif
```

maybe required for linter